packages/honeycomb-opentelemetry-web/test/global-errors-instrumentation.test.ts

```diff
@@ -26,22 +26,35 @@ describe('Global Errors Instrumentation Tests', () => {
 
   describe('when enabled', () => {
     it('should create a span when an error escapes', async () => {
+      const err = new Error('Something happened');
+      err.stack =
+        '' +
+        '  Error: Something happened\n' +
+        '    at baz (filename.js:10:15)\n' +
+        '    at bar (filename.js:6:3)\n' +
+        '    at foo (filename.js:2:3)\n' +
+        '    at (filename.js:13:1)';
```

Comment:
TraceKit seems to be really strict with the format of the stack frames. If this stack frame were formatted as `at filename.js:13:1` then it would be completely ignored. I checked the [regex in the TraceKit package](https://github.com/csnover/TraceKit/blob/7bde6a7a0396e61501d2ab448e0807556ec0e816/tracekit.js#L771) and seems like parenthesis are required.

Reply:
Just wanted to point it out in case it was a cause for concern